lib-multisrc/mangathemesia/src/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesia.kt

```diff
@@ -240,7 +240,7 @@ abstract class MangaThemesia(
 
     override fun mangaDetailsParse(document: Document) = SManga.create().apply {
         document.selectFirst(seriesDetailsSelector)?.let { seriesDetails ->
-            title = seriesDetails.selectFirst(seriesTitleSelector)!!.text()
+            title = seriesDetails.selectFirst(seriesTitleSelector)?.text() ?: ""
```

Comment:
selector should be updated in the source instead

Reply:
yea but secondary title from manga details is usually is just to update the one from popular. so its not really that important if its suddenly not working and it shouldnt interfere with the other details while its broken

Reply:
breaks deeplinks